frontend/packages/kubevirt-plugin/integration-tests/tests/vm.non-admin.scenario.ts

```diff
@@ -62,13 +63,6 @@ describe('Kubevirt non-admin Flow', () => {
   it(
     'ID(CNV-1718) non-admin create project and create/remove vm',
```

Comment:
Since `create project` is moved to `beforeAll`, the test name need to update accordingly.

Reply:
updated the name